### PR TITLE
Feature/perps tests

### PR DESCRIPTION
**Pull request by jmzwar**
Tests for btc, eth, xrp and ltc perps markets.

### CODE REVIEW

File: tomls/omnibus-base-goerli-andromeda/perps/xrp.toml [hunk 1]
```diff
@@ -12,10 +12,10 @@ defaultValue = "1000000"
 defaultValue = "9"
 
 [setting.perpsXrpMakerFeeRatio]
-defaultValue = "0.0007"
+defaultValue = "0.0003"
```
Comment (noisekit) [inline tomls/omnibus-base-goerli-andromeda/perps/xrp.toml:15]: in andromeda branch we also have fixed `pythXrpFeedId` (and for LTC), which are the main culprit.

File: e2e/tests/omnibus-base-goerli-andromeda.toml/Perps_BTC.e2e.js [hunk 2]
```diff
@@ -92,18 +93,63 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
...
+  it('BTC Market Max liquidation parameters are correctly set', async () => {
+    const {
+      maxLiquidationLimitAccumulationMultiplier,
+      maxSecondsInLiquidationWindow,
+      maxLiquidationPd,
+      endorsedLiquidator,
+    } = await PerpsMarketProxy.getMaxLiquidationParameters(200);
+
+    assert.equal(Number(ethers.utils.formatEther(maxLiquidationLimitAccumulationMultiplier)), 0.5);
+    assert.equal(Number(ethers.utils.formatEther(maxSecondsInLiquidationWindow)), 30 / 1e18);
```
Comment (noisekit) [inline e2e/tests/omnibus-base-goerli-andromeda.toml/Perps_BTC.e2e.js:145]:
```suggestion
    assert.equal(maxSecondsInLiquidationWindow.toNumber(), 30);
```

File: tomls/omnibus-base-goerli-andromeda/perps/xrp.toml (hunk 1 above)
Comment (jmzwar) [reply tomls/omnibus-base-goerli-andromeda/perps/xrp.toml:15]: How did that affect maker/taker fees

Comment (noisekit) [reply tomls/omnibus-base-goerli-andromeda/perps/xrp.toml:15]: It does not, but if you change copypaste bugs in xrp config you might as well fix the feed id too.

**Review by noisekit — APPROVED** [review]

**Pull request merged**